bindings/python/CMakeLists.txt

```diff
@@ -32,11 +32,7 @@ endif()
 
 configure_file(${SETUP_PY_IN} ${SETUP_PY})
 
-if( PIP_VERBOSE )
-  set(PIP_INSTALL_VERBOSE_FLAG "--verbose")
-else()
-  set(PIP_INSTALL_VERBOSE_FLAG "")
-endif()
+SET(PIP_OPTIONS "" CACHE STRING "pip options used during the Python bindings install.")
```

Comment:
This might not actually be working 100% correct (hence it is still in draft). But this should give people a clear enough idea of what I'm going for to give feedback.